### ISSUE TITLE
prisma2 init crashing

### ISSUE THREAD

**Issue by AaronNGray** [prisma/prisma2#1068]
I am getting the following happening when I hit any key in any field :-

```
C:\Users\aaron\Tests\Node>prisma2 init hello-prisma

  Connect to your PostgreSQL database
  https://pris.ly/d/postgres-connector

┌─ PostgreSQL database credentials ──────────────────────────────────────────────────────────────────────────────────┐
│                                                                                                                    │
│   Host: localhost                                                                                                  │
│   Port: 5432                                                                                                       │
│ > User: user                                                                                                       │
│   Password:                                                                                                        │
│   Database: postgres                                                                                               │
│   Schema (optional):                                                                                               │
│   [ ] Use SSL                                                                                                      │
│                                                                                                                    │
├─ PostgreSQL database credentials ──────────────────────────────────────────────────────────────────────────────────┤
│                                                                                                                    │
│   URL: postgresql://localhost:5432/admin                                                                           │
│                                                                                                                    │
└────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘

> Connect
< Back (Database options)
The above error occurred in one of your React components:
    in Unknown
    in div (created by Box)
    in Box
    in span (created by Color)
    in Color
    in Unknown (created by Step1PostgresCredentials)
    in div (created by Box)
    in Box (created by BorderBox)
    in div (created by Box)
    in Box (created by BorderBox)
    in BorderBox (created by Step1PostgresCredentials)
    in div (created by Box)
    in Box (created by Step1PostgresCredentials)
    in Step1PostgresCredentials
    in Unknown
    in Unknown
    in TabIndexProvider
    in div (created by Box)
    in Box
    in Unknown
    in App

React will try to recreate this component tree from scratch using the error boundary you provided, App.
```
on Windows, Linux, and MacOSX

**Comment by janpio** [comment]
Thanks for reporting this @AaronNGray - did this work for you before? What command line are you using? Anything special about your Windows that could help reproducing this?

**Comment by AaronNGray** [comment]
Its on all platforms !

**Comment by AaronNGray** [comment]
```
sudo npm install -g prisma2
prisma2 init hello-prisma
```

**Comment by AaronNGray** [comment]
I tried yarn as well and get the same.

**Comment by AaronNGray** [comment]
@janpio "did this work for you before?" - no I have never used Prisma before and was just checking it out.

**Comment by AaronNGray** [comment]
2.0.0-preview018 is crashing
2.0.0-preview017.2 is fine

**Comment by pantharshit00** [comment]
I can confirm this bug. This is an urgent regression that we need to fix.

**Comment by timsuchanek** [comment]
Fixed in latest alpha.
You can try it out with `npm i -g prisma2@alpha`.

**Comment by timsuchanek** [comment]
This now also available in `prisma2@2.0.0-preview018.1`
